packages/server/lib/controllers/connection.controller.ts

```diff
@@ -680,13 +682,14 @@ class ConnectionController {
 
                 if (imported) {
                     updatedConnection = imported;
+                    runHook = true;
```

Comment:
what about `runHook = updatedConnection && updatedConnection.id` so you can simply have `if(runHook)` below

Reply:
Thanks, it is a bit different as runHook depends on if the underlying connection call is initiating the `connectionCreatedHook` call or not. So `runHook` isn't dependent on `updatedConnection && updatedConnection.id `